integration-tests/kyuubi-flink-it/src/test/scala/org/apache/kyuubi/it/flink/FlinkITMixin.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.it.flink
+
+import java.nio.file.Paths
+
+import scala.tools.nsc.io.File
+
+import org.apache.kyuubi._
+
+trait FlinkITMixin {
+
+  val kyuubiHome: String = Utils.getCodeSourceLocation(getClass).split("integration-tests").head
+
+  val jarDir: String = Paths.get(kyuubiHome)
```

Comment:
hadoopClasspath

Reply:
it's actually a folder which contains not only hadoop jars, renamed it to `hadoopJarDir`